README.MD

```diff
@@ -82,129 +79,118 @@ _Handlers_ are used for handling logic when a new packet appears.
 
 #### True connection
 
-When client contacts server at first, it initiates _handshaking_. After it's done, server and
-client will ping each other to ensure connection isn't lost.
+When client interacts with the server at first it initiates _handshaking_. After its completion server and
+client will ping each other in order to ensure connection not being lost.
 
 #### High level packets
 
-In comparison to *Low level*, you can send across the network not only bytes, but also more
-complex entities. To do so, create your own class extending `EnumeratedProto4jPacket` or
-`CallbackProto4jPacket`. All you have to make it working is implement `write(Buffer)` and
-`read(Buffer)` methods and register your packet in `PacketManager` of both sides.
+In contrast to *Low level*, you can send high level packets across the network not only by manipulating raw bytes but also bu using complex entities. To do so, create your own class extending [`EnumeratedProto4jPacket`](./transport/src/main/java/sexy/kostya/proto4j/transport/highlevel/packet/EnumeratedProto4jPacket.java) or
+[`CallbackProto4jPacket`](./transport/src/main/java/sexy/kostya/proto4j/transport/highlevel/packet/CallbackProto4jPacket.java). All you have to do in order to make it working is to implement `write(Buffer)` and
+`read(Buffer)` methods and register your packet in [`PacketManager`](./transport/src/main/java/sexy/kostya/proto4j/transport/highlevel/packet/PacketManager.java) on both sides.
 
-Also, there is new `PacketHandler` class, that works with those packets (and not `Proto4jPacket` ones).
+Also, there is an alternative [`PacketHandler`](./transport/src/main/java/sexy/kostya/proto4j/transport/highlevel/packet/PacketHandler.java) class which works with those packets instead of  [`Proto4jPacket`](.transport/src/main/java/sexy/kostya/proto4j/transport/packet/Proto4jPacket.java)s.
 
 #### Callbacks
 
-It is a common use case to await some packet in response to a sent one. These functionality is
-already implemented at this level. You can specify max awaiting time and handle response the
-way you want. This can be done by sending initial packet using `HighChannel#sendWithCallback` method.
+It is a common scenario to await for some packet responding to the sent ones. These functionality is
+already implemented at this level. You can specify max awaiting time and handle response in the
+way you want. This can be done by sending the initial packet using [`HighChannel`](./transport/src/main/java/sexy/kostya/proto4j/transport/highlevel/HighChannel.java).`sendWithCallback(CallbackProto4jPacket)` method.
 
 ### System properties
 
-The following is a list of system properties with their default values you can specify to
-affect how module works internally. All of time values are in milliseconds.
+The following is a list of system properties which can be used to
+affect the way modules behave internally. All of time values are in specified in milliseconds.
```

Comment:
`All of time values are in milliseconds` or `All of time values specified are in milliseconds`, but your sentence is incorrect.